_data/releasenotes.yml

```diff
@@ -1,6 +1,9 @@
 - title: Swan Lake Release Notes
   url: '#'
   sublinks:
+    - title: Swan Lake 2201.0.0
+      url: /downloads/swan-lake-release-notes/swan-lake-2201-0-0
+      active: swan-lake-2201-0-0
```

Comment:
So these titles and URLs, are we going to keep them in this format? 🤔 Asking cz the version we are using is `2201.0.0` and `Swan Lake` is just the code name. I suppose the title we can easily change to `2201.0.0 (Swan Lake)`. But the others, I'm unsure of what the technical complications will be, if there are any. @anupama-pathirage @sameerajayasoma thoughts?

Reply:
+ Updated as this was already discussed.